vda5050_msgs/msg/ControlPoint.msg

```diff
@@ -1,4 +1,12 @@
-float64 x     # X coordinate described in the world coordinate system.
-float64 y     # Y coordinate described in the world coordinate system.
-float64 weight
-string map_id  # Reference to the ID of the map for the coordinates of the control point.
+float64 x           # X coordinate described in the world coordinate system.
+
+float64 y           # Y coordinate described in the world coordinate system.
+
+float64 orientation # [rad] Range [-pi...pi] Orientation of the AGV on this position of the curve.
+                    # The orientation is in world coordinates.
+                    # When not defined the orientation of the AGV will be tangential to the curve.
```

Comment:
We should probably also update the README to mention that the messages are for 1.1 now. And probably add a "LIMITATIONS" section mentioning that "not defining fields" is not supported currently, which may break some features